loadgen/loadgen.cc

```diff
@@ -748,9 +748,6 @@ void PerformanceSummary::ProcessLatencies() {
     assert(lp.percentile < 1.0);
     lp.value = pr.latencies[sample_count * lp.percentile];
   }
-
-  // Clear latencies since we are done processing them.
-  pr.latencies = std::vector<QuerySampleLatency>();
```

Comment:
Removed those lines to use `pr.latencies` to report through `PerformanceSummary`, i.e. `sut->ReportLatencyResults(base_perf_summary.pr.latencies);`